### DISCUSSION TITLE
Why image previews don't it work in PuTTY?

### DISCUSSION THREAD

**Discussion by adrianstaniec** [sxyazi/yazi#1664] [1. Q&A]
I am connecting from Windows 10 machine via putty to and Ubuntu 22.04.
I don't see any image previews.
I just wanted to understand why and what would be needed to make it work.
It would be super handy.

**Comment by sxyazi** [comment]
PuTTY does not support any image protocol (IIP, Sixel, Kitty), you need a terminal that has the image preview feature. There's a list of all terminals that support display images https://yazi-rs.github.io/docs/image-preview
